OpenRA.Renderer.Sdl2/Sdl2GraphicsDevice.cs

```diff
@@ -308,6 +308,34 @@ public void SetLineWidth(float width)
 			ErrorHandler.CheckGlError();
 		}
 
+		public Bitmap TakeScreenshot(Rectangle? area)
+		{
+			var rect = new Rectangle(Point.Empty, size);
+			if (area.HasValue)
```

Comment:
We don't use this capability for capturing a non-screen-sized area, nor do we really have a use case for it.